large_data/README.md

```diff
@@ -0,0 +1,67 @@
+# Reference Implementation for Utilizing Large Pages for Data
+This directory contains a simple reference implementation for automating the process
+of utilizing explicit huge pages for Data.
+
+
+# APIs
+```
+* bool iodlr_hp_enabled()
+  Check whether explict huge pages is enabled on the system
+  Returns true if Huge Pages is enabled on the system
+* size_t iodlr_get_hp_size()
+  Obtains the size of the explicit huge pages (2MB or 1GB) in bytes
+* void * iodlr_allocate(size_t s, size_t pgsz)
+  mmap size bytes with pgsz
+* void iodlr_deallocate(char *d, size_t s)
+  unmap
+* int MapStaticCodeToLargePages(hotstart, hotend)
+  Map region from hotstart to hotend to 2MB pages
+  Returns -1 if an error occurs while mapping
```

Comment:
```suggestion
```

Reply:
Thanks for reviewing the documentation as well!